plus/async/sleep.js

```diff
@@ -4,4 +4,16 @@
  * @param {number} sec the seconds to wait
  * @returns {Promise} the promise which resolves when wait if over
  *                    or rejects if parameter is not correct
- */
\ No newline at end of file
+ */
+
+module.exports = async function sleep(sec) {
+  if (typeof sec !== "number") {
```

Comment:
There is a function in JS, which can determine whether a value is not a number. Could you please find it?